[2025-03-19T18:43:09.987+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-19T18:43:10.045+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_Api_v1.get_name scheduled__2024-04-04T00:00:00+00:00 [queued]>
[2025-03-19T18:43:10.061+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_Api_v1.get_name scheduled__2024-04-04T00:00:00+00:00 [queued]>
[2025-03-19T18:43:10.062+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-03-19T18:43:10.086+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2024-04-04 00:00:00+00:00
[2025-03-19T18:43:10.099+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=2227) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-19T18:43:10.104+0000] {standard_task_runner.py:72} INFO - Started process 2242 to run task
[2025-03-19T18:43:10.101+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_Api_v1', 'get_name', 'scheduled__2024-04-04T00:00:00+00:00', '--job-id', '575', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_taskflow_api.py', '--cfg-path', '/tmp/tmpnsw1hj08']
[2025-03-19T18:43:10.108+0000] {standard_task_runner.py:105} INFO - Job 575: Subtask get_name
[2025-03-19T18:43:10.238+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_Api_v1.get_name scheduled__2024-04-04T00:00:00+00:00 [running]> on host 63fd4b143e0a
[2025-03-19T18:43:10.419+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_Api_v1' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2024-04-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-04-04T00:00:00+00:00'
[2025-03-19T18:43:10.423+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-19T18:43:10.428+0000] {python.py:240} INFO - Done. Returned value was: Jerry
[2025-03-19T18:43:10.481+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-19T18:43:10.483+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_Api_v1, task_id=get_name, run_id=scheduled__2024-04-04T00:00:00+00:00, execution_date=20240404T000000, start_date=20250319T184310, end_date=20250319T184310
[2025-03-19T18:43:10.731+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-19T18:43:10.831+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-19T18:43:10.847+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
